src/Avalonia.Base/AvaloniaObject.cs

```diff
@@ -304,6 +305,14 @@ public bool IsSet(AvaloniaProperty property)
 
             VerifyAccess();
 
+            var description = GetDescription(source);
+
+            if (priority == BindingPriority.LocalValue)
```

Comment:
That's a holdover from when I was trying to fix the flickering. Sorry about that. Just fixed it.